src/bbs2gh/RsaSha256Util.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Reflection;
+using System.Security.Cryptography;
+using Renci.SshNet;
+using Renci.SshNet.Common;
+using Renci.SshNet.Security;
+using Renci.SshNet.Security.Cryptography;
+using Renci.SshNet.Security.Cryptography.Ciphers;
+
+namespace OctoshiftCLI.BbsToGithub;
+
+// workaround for RSA keys on Ubuntu 22.04
+// https://github.com/sshnet/SSH.NET/issues/825#issuecomment-1139440419
+
+public class RsaWithSha256SignatureKey : RsaKey
```

Comment:
As I already mentioned this seems like a quick workaround rather than a production ready code and really hard to maintain! If we really want to go this route (I wouldn't recommend it) we need to spend some time and understand  what this is doing and probably re-write it and add unit tests, etc.

Reply:
I re-wrote it to be a bit simpler (IMO), but more importantly it only applies this behavior for ssh key's that are RSA (i.e. it won't break ed25519 or other key formats).

I thought about how we could unit test it, and ended up at the opinion that the amount of effort to unit test this chunk of code isn't worth it (it would be quite difficult, and probably require file system access). Hopefully at some point in the near future SSH.Net will release a new version and make this code obsolete.